src/algebra/group/with_one/basic.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2018 Mario Carneiro. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Johan Commelin
+-/
+import algebra.group.with_one.defs
+import algebra.hom.equiv.basic
+
+/-!
+# Adjoining a zero/one to semigroups and related algebraic structures
```

Comment:
This docstring is inaccurate, it's just a duplicate of the file or was split from.